source/Outfit.cpp

```diff
@@ -109,6 +109,7 @@ namespace {
 		{"discharge resistance energy", 0.},
 		{"discharge resistance fuel", 0.},
 		{"discharge resistance heat", 0.},
+		{"drag reduction", -0.99},
```

Comment:
Any reason why you added this here? Seems like a better fit with the other -.99 limits imo